cookiecutter-xblock/{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -0,0 +1,90 @@
+[tox]
+envlist = py38-django{32,40}, quality, docs, pii_check
+skipsdist = true
+
+[doc8]
+; D001 = Line too long
+ignore = D001
+
+[pycodestyle]
+exclude = .git,.tox,migrations
+max-line-length = 120
+
+[pydocstyle]
+; D101 = Missing docstring in public class
+; D200 = One-line docstring should fit on one line with quotes
+; D203 = 1 blank line required before class docstring
+; D212 = Multi-line docstring summary should start at the first line
+; D215 = Section underline is over-indented (numpy style)
+; D404 = First word of the docstring should not be This (numpy style)
+; D405 = Section name should be properly capitalized (numpy style)
+; D406 = Section name should end with a newline (numpy style)
+; D407 = Missing dashed underline after section (numpy style)
+; D408 = Section underline should be in the line following the section’s name (numpy style)
```

Comment:
This apostrophe in `section's` is a non-ascii character. Please use simple apostrophe from the keyboard as we've recently removed non-ascii characters from current repos.

Reply:
Didn't we add any new check to avoid adding any such values in future?

Reply:
Removed all non-ascii characters similar to the one pointed out above.